Gatekeeper/Dockerfile

```diff
@@ -1,20 +1,15 @@
-FROM microsoft/dotnet:2.1-aspnetcore-runtime AS base
+FROM microsoft/dotnet:2.1-sdk AS build-env
 WORKDIR /app
-EXPOSE 57887
-EXPOSE 44338
 
-FROM microsoft/dotnet:2.1-sdk AS build
-WORKDIR /src
-COPY ["Gatekeeper/Gatekeeper.csproj", "Gatekeeper/"]
-RUN dotnet restore "Gatekeeper/Gatekeeper.csproj"
-COPY . .
-WORKDIR "/src/Gatekeeper"
-RUN dotnet build "Gatekeeper.csproj" -c Release -o /app
+COPY *.csproj ./
+RUN dotnet restore
 
-FROM build AS publish
-RUN dotnet publish "Gatekeeper.csproj" -c Release -o /app
+COPY . ./
+RUN dotnet tool install -g Microsoft.Web.LibraryManager.Cli
+RUN /root/.dotnet/tools/libman restore
+RUN dotnet publish -c Release -o out
```

Comment:
Does this build the release copy in the docker container then? If so this should be built on Travis, then the artifacts copied across.

Reply:
They're built in the build-env container which is thrown away once the image is created.  The only stuff in the final image is from line 12 onwards, where it copies the compiled output from build-env.

Reply:
👍